robot/Cumulus/resources/GiftEntryPageObject.py

```diff
@@ -122,9 +125,9 @@ def enter_value_in_field(self,**kwargs):
                 self.salesforce._populate_field(locator, value)   
 
     @capture_screenshot_on_error
-    def select_object_group_field(self,object_group,field):
-        """Select the specified field under specified object group 
-           to add the field to gift entry form and verify field is added"""
+    def object_group_field_action(self,action,object_group,field):
```

Comment:
the name of the keyword can be modified to be more readable like "Perform_action_on_object_group"

Reply:
done